File: beecrowd/1_Beginner/1101_Sequence_of_Numbers_and_sum.c
#include <stdio.h>
#include <math.h>

int main()
{

    int i=0;
    while(1)
    {
        int x,y;
        scanf("%d %d",&x,&y);

        if(x<=0 || y<=0)
        {
            break;
        }
        else
        {
            if(x>y)
            {
                int temp = x;
                x = y;
                y = temp;
            }

            int sum=0;
            for(int j=x; j<=y; j++)
            {
                    sum += j;
                    printf("%d ", j);
            }
            printf("Sum=%d\n",sum);
        }

    }

    return 0;
}

File: beecrowd/1_Beginner/Python/1101_Sequence_of_Numbers_and_sum.py
i=int(0)
while(True):
    row = list(map(int, input().split()))
    x = int(row[0])
    y = int(row[1])
    # if i==0:
    #     print()
    #     i=1

    if x<=0 or y<=0:
        break
    else:
        if x>y:
            temp = x
            x = y
            y = temp

        sum= int(0)

        for j in range(x,y+1):
                sum += j
                print(str(j), end =" ")
        print("Sum=" + str(sum))
